docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -224,30 +224,37 @@ const Warning = ({ children }) => {
   );
 };
 
-const CodeReferenceLink = (props: { filePath: string }) => {
+const CodeReferenceLink = (props: { filePath: string, format?: string }) => {
```

Comment:
maybe could just have a bool `is_inline` to simplify the edge case handling

Reply:
[1] `props` may not be necessary and instead you could do:
```
const CodeReferenceLink = ({ filePath: string, format?: string, children: JSX.Element }) => {
```

Reply:
I added `isInline`, but the latter change causes errors for reason I don't understand-- when you remove `props` you can't have types for the member properties anymore-- it complains that `filePath` is not defined. So I removed `props` but also removed the types, which is consistent with the rest of the components on the page.